input/vulnerabilities/APK_duplicate_file.json

```diff
@@ -27,5 +27,11 @@
     "verge-android-4.3" : {
       "url" : "http://www.theverge.com/2013/7/24/4550234/android-4-3-announcement"
     }
-  }
+  },
+  "Surface": ["local", "app"],
+  "Vector": ["insufficient-standards-verification"],
+  "Target": ["apps"],
+  "Channel": ["app-execution"],
+  "Condition": ["affected-app-installed", "unknown-source-install-allowed"],
+  "Privilege": ["user", "modify-apps"]
```

Comment:
I think these vulnerabilities grant "system" as you can modify an existing "system" privilege app to have new functionality without changing the signature.

Reply:
Yes - will add this (`system` was only added to the documentation halfway through so some of the earlier vulnerabilities to be tagged may be missing it)